adobe_vipm/flows/airtable.py

```diff
@@ -429,3 +511,108 @@ def get_prices_for_3yc_skus(product_id, currency, start_date, skus):
         if item.sku not in prices:
             prices[item.sku] = item.unit_pp
     return prices
+
+
+def create_global_customer_agreement_deployment(product_id, agreement_deployment):
```

Comment:
Should we update these create functions to be batch saves with a dictionary list? What do you think?

Reply:
I agree with you! I have updated the method. Thank you! :-)